packages/suite/src/components/suite/index.tsx

```diff
@@ -100,6 +101,8 @@ export {
     PinMatrix,
     CollapsibleBox,
     DeviceMatrixExplanation,
+    useNavHook,
```

Comment:
This file is for components, not hooks

Reply:
bc4b072ebba461e68470f831178ea69760baf51c